### DISCUSSION TITLE
telegram

### DISCUSSION THREAD

**Discussion by Adivarjunior** [arendst/Tasmota#21092] [Q&A]
Ola pessoal. Gostria que receber noitificaçoes do telegram quando mudar o estado das portas do esp8266. Alguem sabe como fazer essa configuraçoa?

**Comment by lalo-uy** [comment]
Vc tein que colocar rules en Power on e Power off que envien o telegram.
O foro en en ingles , por favor.

You need to set rules on Power on and off that sen the Telegram
